test/uk/gov/hmrc/merchandiseinbaggage/content/PreviousDeclarationDetailsContentSpec.scala

```diff
@@ -21,34 +21,36 @@ import uk.gov.hmrc.merchandiseinbaggage.CoreTestData
 import uk.gov.hmrc.merchandiseinbaggage.model.api.JourneyOnFoot
 import uk.gov.hmrc.merchandiseinbaggage.model.api.JourneyTypes.Amend
 import uk.gov.hmrc.merchandiseinbaggage.smoketests.pages.PreviousDeclarationDetailsPage
-import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub.{givenAPaymentCalculation, givenPersistedDeclarationIsFound}
+import uk.gov.hmrc.merchandiseinbaggage.stubs.MibBackendStub
 
 import java.time.LocalDate
 
 class PreviousDeclarationDetailsContentSpec extends PreviousDeclarationDetailsPage with CoreTestData {
 
+  private val mibStub = app.injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks